csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,284 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: 'job!=""',
+    groupLabels: ['job', 'project_id'],
+    instanceLabels: [],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization',
+    },
+    signals: {
+      instanceCount: {
+        name: 'Instance count',
+        description: 'Number of VM instances',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'count(sum by (instance_name) (stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      systemProblemCount: {
+        name: 'VM Availability',
+        description: 'Measure of availability of virtual machines',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'count(sum by(instance_id)(stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{%(queriesSelector)s}))',
+            legendCustomTemplate: '',
+          },
+        },
+      },
+      top5CpuUtilization: {
+        name: 'Top 5 Instances by CPU Utilitization',
+        description: 'Fractional utilization of allocated CPU on an instance',
+        type: 'gauge',
+        aggFunction: 'sum',
+        sources: {
+          stackdriver: {
+            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name', 'job', 'project_id'],
+            exprWrappers: [['topk(5, 100*', ')']],
+          },
+        },
+      },
+
+      top5SystemProblem: {
+        name: 'Top 5 Instances by System problem',
+        description: 'List of top 5 instances with system problems.',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'topk(5, sum by (instance_name, job, project_id) (increase(stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{%(queriesSelector)s}[$__range])))',
+            legendCustomTemplate: '',
```

Comment:
You've used `exprWrappers` on some other signals. Is the default aggregation not suitable for this signal? Could you use gauge, and a `rangeFunction` of `increase` for this?

Same feedback/question for the next few signals as well.

Reply:
@rgeyer If I configure it like this 
```
type: 'gauge',
        rangeFunction: 'increase',
        sources: {
          stackdriver: {
            expr: 'stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{%(queriesSelector)s}',
            legendCustomTemplate: '',
            aggKeepLabels: ['instance_name'],
            exprWrappers: [['topk(5, ', ')']],
          },
        },
```
it results in: `topk(5, 
  avg by (job,project_id,instance_name) (  stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{job!="",job=~"$job",project_id=~"$project_id"}))`
which lacks the increase. I also tried adding the `increase` in the stackdriver object, but it's the same.

Reply:
you have type: gauge here. For gauge range functions are not applied as doesn't make sense. 

increase can be added by setting signal type to `counter` and also setting `rateFunction:increase` instead of default's `rate`.

Reply:
This metric(**stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count**) is a count, so I think we should use type: 'counter' with a rate inside, like this:
```
topk(5, 
  avg by (job,project_id,instance_name) (
  rate(stackdriver_gce_instance_compute_googleapis_com_guest_system_problem_count{job!="",job=~"$job",project_id=~"$project_id"}[$__rate_interval])
)
)
```
but the results aren't looking right.